packages/launchpad/cypress/fixtures/scaffolded-files.json

```diff
@@ -0,0 +1,127 @@
+{
+    "component": [
+       {
+           "status": "changes",
+           "description": "Merge this code with your existing config file.",
+           "file": {
+               "id": "1",
+               "relative": "cypress.config.ts",
+               "absolute": "/Users/rachel/Documents/GitHub/cypress/packages/launchpad/cypress.config.ts",
+               "contents": "Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...",
+               "fileExtension": ".ts"
+           }
+       },
+       {
+           "status": "valid",
+           "description": "An example fixture for data imported into your Cypress tests, such as `cy.intercept()`.",
```

Comment:
> If there's a preference here to do a system test instead, lmk and I'll be happy to update!

Normally we don't add a system test if we can avoid it, since they take so long to run. I'd say E2E tests in cypress/e2e are fine for this kind of fix.